lib/vagrant/environment.rb

```diff
@@ -123,6 +126,7 @@ def initialize(opts=nil)
       @data_dir   = @home_path.join("data")
       @gems_path  = Vagrant::Bundler.instance.plugin_gem_path
       @tmp_path   = @home_path.join("tmp")
+      @aliases_path = @home_path.join("aliases")
```

Comment:
It would be nice to allow ENV var override for this, something like `ENV.fetch("VAGRANT_ALIAS_FILE", @home_path.join("aliases"))`